mysite/chat/serializers.py

```diff
@@ -281,6 +281,58 @@ def get_progress(self, obj):
         return progress
 
 
+class AddUnitByChatStepSerializer(serializers.ModelSerializer):
+    """
+    Serializer for Message, used when user in "Add unit by chat" chat to serialize messages for Progress
```

Comment:
why only 80 chars? I thought we decided to extend it to 120 chars, cause all of us are using wide displays and 80 chars will be less then half of visible screen. Does it make any sense?